conf/machine/arrakis-pico.conf

```diff
@@ -0,0 +1,66 @@
+require conf/machine/genericx86-64.conf
+
+# additional kernel config
+OMNECT_KERNEL_SRC_URI:append = " \
+    file://crc32c-intel_module.cfg \
+    file://disable_wlan.cfg \
+    file://intel-lpc_ich_module.cfg \
+    file://intel-mei_module.cfg \
+    file://intel-vbtn_module.cfg \
+    file://pc-speaker.cfg \
+    file://tcg_tpm_2.0.cfg \
+"
+# default kernel parameter
+APPEND = "console=tty0"
+# module autoload
+KERNEL_MODULE_AUTOLOAD = "mei"
+
+# bootloader
+PREFERRED_PROVIDER_virtual/bootloader = "grub-efi"
+# don't create grub-efi in rootfs
+EFI_PROVIDER = " "
+
+# swupdate has a hard dependency to libubootenv (in swupdate_vars.(c|h)).
+# so we can not get rid of it, but we can get rid of a uboot boot script.
+DEPENDS:remove:pn-libubootenv = "u-boot-default-script"
+
+# initramfs type
+OMNECT_INITRAMFS_FSTYPE = "cpio.gz"
+
+# image generation
+WKS_FILE = "omnect-os.grub.wks.in"
+# no differentiation for now between release an devel images concerning rootfs
+# size
+OMNECT_PART_SIZE_ROOTFS ?= "${@bb.utils.contains('OMNECT_RELEASE_IMAGE', '1', '819200', '819200', d)}"
+# where to find initial grubenv
+WICVARS:append = " OMNECT_GRUBENV_FILE"
+OMNECT_GRUBENV_FILE = "${LAYERDIR_omnect}/files/grubenv"
+OMNECT_ROOT_DEVICE = "/dev/nvme0n1"
+
+# adu compatibility id
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1"
```

Comment:
@JanZachmann btw: an omnect variable without `OMNECT_` prefix. fix it as part of this PR? or as part of #384?

Reply:
from context point of view #384 but since there are also changes in omnect-os needed I would go for this PR and https://github.com/omnect/omnect-os/pull/426. Or a operational task if you don't want

Reply:
renamed `ADU_*` -> `OMNECT_ADU_*`